packages/amazonq/src/lsp/lspInstaller.ts

```diff
@@ -22,21 +27,27 @@ export class AmazonQLspInstaller extends BaseLspInstaller.BaseLspInstaller<
     protected override async postInstall(assetDirectory: string): Promise<void> {
         const resourcePaths = this.resourcePaths(assetDirectory)
         await fs.chmod(resourcePaths.node, 0o755)
+        if (await fs.exists(resourcePaths.rg)) {
+            await fs.chmod(resourcePaths.rg, 0o755)
+        }
     }
 
     protected override resourcePaths(assetDirectory?: string): AmazonQResourcePaths {
         if (!assetDirectory) {
             return {
                 lsp: this.config.path ?? '',
                 node: getNodeExecutableName(),
+                rg: `ripgrep/${getRgExecutableName()}`,
                 ui: this.config.ui ?? '',
             }
         }
 
         const nodePath = path.join(assetDirectory, `servers/${getNodeExecutableName()}`)
+        const rgPath = path.join(assetDirectory, `servers/ripgrep/${getRgExecutableName()}`)
         return {
             lsp: path.join(assetDirectory, 'servers/aws-lsp-codewhisperer.js'),
             node: nodePath,
+            rg: rgPath,
```

Comment:
can you call this ripGrep instead of rg? in 6 months when someone looks at this code it might not be obvious what it is

Reply:
sure